pysingfel/detector/epix10k.py

```diff
@@ -1,16 +1,118 @@
 import re
 import six
-
+import numpy as np
 from pysingfel.util import deprecated
-from .lcls import LCLSDetector
+from .autoranging import AutoRangingDetector
+
+class Epix10kDetector(AutoRangingDetector):
+    def __init__(self, cameraConfig, *args, **kwargs):
+        super(Epix10kDetector, self).__init__(cameraConfig=cameraConfig, *args, **kwargs)
```

Comment:
In this line (and this line only), I think you should have
`*args, cameraConfig=cameraConfig, **kwargs`
as non-keyword arguments (`*args`) cannot come after keyword arguments (`cameraConfig` and `**kwargs`).
That would only crash if some unnamed arguments are provided when creating the Epix10kDetector, tho.

Reply:
line 9?

Reply:
line 9? Below is the error message after changing the order, though.
Traceback (most recent call last):
  File "Test-ePix10ka2M-Detector.py", line 28, in <module>
    det = ps.Epix10kDetector(geom=ROOT_DIR+'/input/lcls/xcsx35617/Epix10ka2M::CalibV1/XcsEndstation.0:Epix10ka2M.0/geometry/0-end.data',run_num=0,beam=beam,cameraConfig='fixedMedium')
  File "/reg/neh/home5/iris/Software/pysingfel/pysingfel/detector/epix10k.py", line 10, in __init__
    super(Epix10kDetector, self).__init__(**args, cameraConfig=cameraConfig, **kwargs)
TypeError: __init__() argument after ** must be a mapping, not tuple

Reply:
line 9?

Reply:
line 9? What about line 8?

Reply:
Just line 9. In line 8, it's not a keyword argument. Keyword arguments are only when you use the `=`.

Reply:
fixed